app/frontend/src/features/friends/FriendsPage.tsx

```diff
@@ -0,0 +1,27 @@
+import { Grid } from "@material-ui/core";
+import React from "react";
+import PageTitle from "../../components/PageTitle";
+import FriendList from "./FriendList";
+import FriendRequests from "./FriendRequests";
+import FriendRequestsSent from "./FriendRequestsSent";
+
+function FriendsPage() {
+  return (
+    <>
+      <PageTitle>Friends</PageTitle>
+      <Grid container spacing={2}>
+        <Grid item xs={12} md={6}>
+          <FriendList />
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <FriendRequests />
+        </Grid>
+        <Grid item xs={12} md={6}>
+          <FriendRequestsSent />
```

Comment:
Are these supposed to be md=4?

Reply:
I did `md=6` here as the [WIP hi-fi desktop wireframes here](https://www.figma.com/file/wYr6z7KQpoIIN0A97jD7kf/Couchers-Platform-Design?node-id=680%3A1) seem to suggest we want to show two tiles per row. So maybe we should stick with it? :shrug: